src/components/custom/_signup-affiliate.js

```diff
@@ -0,0 +1,217 @@
+import React, { useState } from 'react'
+import styled from 'styled-components'
+import PropTypes from 'prop-types'
+import { navigate } from 'gatsby'
+import AgreementLabel from './_agreement-label'
+import { Input, Button } from 'components/form'
+import { Header, Text, LinkText } from 'components/elements'
+import { localize } from 'components/localization'
+import device from 'themes/device.js'
+
+const SignupContent = styled.div`
+    width: 48.4rem;
+    display: flex;
+    justify-content: flex-start;
+    flex-direction: column;
+    padding: 4rem;
+    background-color: var(--color-white);
+    border-radius: 0.6rem;
+    box-shadow: 0 1.6rem 2rem 0 rgba(0, 0, 0, 0.1);
+
+    @media ${device.tablet} {
+        width: 53rem;
+    }
+    @media ${device.mobileL} {
+        width: 100%;
+        padding: 6rem 2rem;
+    }
+`
+const StyledHeader = styled(Header)`
+    @media ${device.tabletL} {
+        font-size: 3rem;
+        margin-bottom: 3rem;
+    }
+`
+
+const SubTitle = styled(Text)`
```

Comment:
This should be replaced with the use of `Header` along with responsive style props.

Reply:
You mentioned this is replaced but I still see `Text` used quite a fair bit of times. Please use `Header` with `type` and `as` instead.

Reply:
Thanks for getting attention on that. I re-checked components created by me so far and replaced `Text` components everywhere.